seed-db/scripts/index.js

```diff
@@ -6,7 +6,6 @@ const { generateRegulations } = require("./regulation");
 const { generateProgrammeDegrees } = require("./programme-degree");
```

Comment:
Line 5 needs to be removed - docker-compose gives an error because regulation.js doesn't exist

Reply:
done